File: src/Lesson4/java/Ex5.java
package Lesson4.java;
import java.util.Scanner;

public class Ex5 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter the lines from homework #5:");
        System.out.println("Enter the first line: ");
        String a = scanner.nextLine();

        System.out.println("Enter the second line: ");
        String b = scanner.nextLine();

        System.out.println("Result: ");

        String s1 = a.toLowerCase();
        String s2 = b.toLowerCase();

        if (s1.compareTo(s2) < 0) {
            System.out.println(-1);
        } else if (s1.compareTo(s2) > 0) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }

        scanner.close();
    }
}

File: src/Lesson4/forkotlin/Ex5.kt
package Lesson4.forkotlin

fun main() {
    println("Enter the lines from homework #5:")
    println("Enter the first line: ")
    val a = readln()
    println("Enter the second line: ")
    val b = readln()
    println("Result: ")
    val s1 = a.lowercase()
    val s2 = b.lowercase()

    if (s1 < s2) {
        println(-1)
    } else if (s1 > s2) {
        println(1)
    } else {
        println(0)
    }
}